tools/mtmd/clip.cpp

```diff
@@ -4790,8 +4498,8 @@ int clip_n_mmproj_embd(const struct clip_ctx * ctx) {
         case PROJECTOR_TYPE_LFM2:
         case PROJECTOR_TYPE_KIMIVL:
             return ctx->model.mm_2_w->ne[1];
-        case PROJECTOR_TYPE_COGVLM:
-            return ctx->model.mm_4h_to_h_w->ne[1];
+        case PROJECTOR_TYPE_JANUS_PRO:
```

Comment:
Maybe better to move this after `case PROJECTOR_TYPE_QWEN25VL` ?

We can read the shape of bias tensor instead of the weight, they should be the same